app/javascript/components/mentoring/session/CancelledRequestModal.tsx

```diff
@@ -0,0 +1,47 @@
+import { Modal } from '@/components/modals'
+import React from 'react'
+import { Links } from '../Session'
+
+export function CancelledRequestModal({
+  open,
+  onClose,
+  links,
+  isLocked,
+}: {
+  open: boolean
+  onClose: () => void
+  links: Links
+  isLocked: boolean
+}): JSX.Element {
+  return (
+    <Modal
+      onClose={onClose}
+      style={{ content: { width: '740px' } }}
+      open={open}
+      shouldCloseOnEsc={false}
+      shouldCloseOnOverlayClick={false}
+      aria={{
+        modal: true,
+        describedby: 'cancelled-mentoring-request-description',
+      }}
+    >
+      <h3
+        id="cancelled-mentoring-request-description"
+        className="text-h3 mb-16 text-center"
+      >
+        The student has cancelled this mentoring request. <br />
```

Comment:
@iHiD Please review the wording and the structure here. Perhaps it would be better to place the 'Sorry...' text inside a div and keep only 'The student..." text in the h3. Having a line break within the header looks odd. I just wanted to see how the text would appear on a new line.